NKI_data.Rmd

```diff
@@ -0,0 +1 @@
+Gathering Sustainability Data on funds for Retail Investors is difficult
```

Comment:
Sounds like a meaningful problem and if you have a gist of solution, you should go ahead and share it.